apps/speed.c

```diff
@@ -327,6 +328,8 @@ const OPTIONS speed_options[] = {
      "Run benchmarks for pnum seconds"},
     {"bytes", OPT_BYTES, 'p',
      "Run cipher, digest and rand benchmarks on pnum bytes"},
+    {"aead", OPT_AEAD, '-',
```

Comment:
this flag must be used in conjonction with -evp , isn't it ?

Reply:
Not only that, but with cipher. I mean -evp can denote digest as well. Yeah, maybe sanity check with an error message would be appropriate...

Reply:
I suggest to take inspiration on the 'mb' option, a few lines above ::
```
     {"mb", OPT_MB, '-',
      "Enable (tls1.1) multi-block mode on evp_cipher requested with -evp"},
```

Reply:
Like that?